modular_skyrat/modules/titlescreen/code/titlescreen_html.dm

```diff
@@ -0,0 +1,101 @@
+
+GLOBAL_LIST_EMPTY(startup_messages)
+// FOR MOR INFO ON HTML CUSTOMISATION, SEE: https://github.com/Skyrat-SS13/Skyrat-tg/pull/4783
+
+#define MAX_STARTUP_MESSAGES 27
+
+/mob/dead/new_player/proc/get_lobby_html()
+	var/dat = GLOB.lobby_html
+	if(SSticker.current_state == GAME_STATE_STARTUP)
+		dat += {"<img src="titlescreen.gif" class="fone" alt="">"}
+		dat += {"
+		<div class="container_terminal">
+			<p class="menu_b">SYSTEMS INITIALIZING:</p>
+		"}
```

Comment:
Why can't those two things be in the same `+=` operation?

Also, that titlescreen.gif should also probably be a define, so that it can easily be changed later on.